examples/bert_app.py

```diff
@@ -48,6 +48,9 @@
 
 optimizer = fluid.optimizer.Adam(learning_rate=learning_rate)
 optimizer = fleet.distributed_optimizer(optimizer, dist_strategy)
+clip_norm_thres = 1.0
```

Comment:
I think clip does not work if you do not put this into Adam initilization

Reply:
It works, but it is not recommended. I'll add it to Adam.